WCF/Shared/RequestTrackingTelemetryModule.cs

```diff
@@ -44,7 +44,7 @@ void IWcfTelemetryModule.OnBeginRequest(IOperationContext operation)
             var httpHeaders = operation.GetHttpRequestHeaders();
             if ( httpHeaders != null )
             {
-                telemetry.HttpMethod = httpHeaders.Method;
+                telemetry.Properties["httpMethod"] = httpHeaders.Method;
```

Comment:
we append the method to the operation name in Web. Do not send it separately. It's ok if you want to send it though